lib/base/string.hpp

```diff
@@ -23,6 +24,13 @@ class Value;
  */
 class String
 {
+private:
+#if BOOST_VERSION >= 108100
+	typedef boost::beast::string_view BeastStringView;
+#else
+	typedef boost::string_view BeastStringView;
+#endif
+
```

Comment:
[`boost::beast::string_view` already existed before](https://www.boost.org/doc/libs/1_66_0/libs/beast/doc/html/beast/ref/boost__beast__string_view.html) and [1.81 only changed the type it's aliased to](https://github.com/boostorg/beast/commit/9d23bec2bc523223a377582302bd28d04cb514ed). So you shouldn't need `#if` here and just always use it.